lib/bitex_bot/models/opening_flow.rb

```diff
@@ -125,8 +126,8 @@ def self.open_position?(transaction)
       open_position_class.find_by_transaction_id(transaction.id)
     end
 
-    def self.btc_specie?(transaction)
-      transaction.specie == :btc
+    def self.btc_usd_orderbook?(transaction)
+      transaction.orderbook == :btc_usd
```

Comment:
This is comparing against a hardcoded orderbook, we should compare it against BitexBot::Settings.bitex.orderbook

Reply:
👍